src/main/java/com/mongodb/spark/sql/connector/config/WriteConfig.java

```diff
@@ -87,16 +86,28 @@ public enum ConvertJson {
       this.value = operationType;
     }
 
-    static ConvertJson fromString(final String jsonType) {
-      if (jsonType.equalsIgnoreCase(TRUE)) {
+    static ConvertJson fromString(final String operationType) {
```

Comment:
[optional]
Not that I understand how the previous name "jsonType" was a good description for a value of the `convertJson` option, but neither do I understand why "operationType" is a good description for that. Wouldn't `operationType` or even just `v` be more relevant?

Reply:
I just used it as its the same name in the constructor.  Will update.